packages/block-editor/src/phone-number/attributes.js

```diff
@@ -0,0 +1,40 @@
+export default {
+	clientId: {
+		type: 'string',
+		default: null,
+	},
+	country: {
+		type: 'string',
+		default: '',
+	},
+	flag: {
+		type: 'boolean',
+		default: false,
+	},
+	label: {
+		type: 'string',
+		default: 'Phone Number',
```

Comment:
This makes the block label have an actual value when placed instead of having a placeholder text.

Reply:
Changed to placeholder